services/user_api/queries.js

```diff
@@ -0,0 +1,82 @@
+const Pool = require('pg').Pool
+const pool = new Pool({
+  user: 'admin',
+  host: 'localhost',
+  database: 'CS3219',
+  password: 'password',
+  port: 5432,
+})
+
+
+const getUsers = (request, response) => {
+    pool.query('SELECT * FROM users ORDER BY userId ASC', (error, results) => {
+      if (error) {
+        throw error
+      }
+      response.status(200).json(results.rows)
+    })
+  }
+
+const getUserById = (request, response) => {
+const id = parseInt(request.params.id)
+
+pool.query('SELECT * FROM users WHERE userId = $1', [id], (error, results) => {
+    if (error) {
+    throw error
+    }
+    response.status(200).json(results.rows)
+})
+}
+
+
+const createUser = (request, response) => {
+const { username, password, email, createdDateTime} = request.body
+
+pool.query(
+    'INSERT INTO users (username, password, email, createdDateTime) VALUES ($1, $2, $3, $4) RETURNING *',
+    [username, password, email, createdDateTime],
+    (error, results) => {
+      if (error) {
+        throw error;
+      }
+      response.status(201).send(`User added!`);
```

Comment:
There's a typo on the response status

Reply:
If you meant the response status of 201, I think it's correct instead of status 200. The response status of 201 seems to be more explicit in defining the intention here.

> The HTTP 201 Created success status response code indicates that the request has succeeded and has led to the creation of a resource.

>The HTTP 200 OK success status response code indicates that the request has succeeded